src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age !== undefined && <p className="Person__age">I am {person.age}</p>}
```

Comment:
Specifically in this case, it is simpler to write `{person.age`... , but I wrote `{person.age !== undefined`... in case, in particular, the age will be equal to zero. In this case, `{person.age`... will not work properly